idd/Energy+.idd.in

```diff
@@ -20283,7 +20274,7 @@ SteamEquipment,
        \ip-units Btu/h-ft2
   N3 , \field Power per Person
        \type real
-       \minimum 0
+       \minimum
```

Comment:
Blah. Something happened to my Notepad++ install, so I reinstalled a few days back. When I was fixing diffs in this I realized that my whitespace elimination macro wasn't working. This must have gotten deleted while I was testing that.

I'll put it back.